wemake_python_styleguide/violations/consistency.py

```diff
@@ -1141,3 +1143,34 @@ class UselessExceptCaseViolation(ASTViolation):
 
     error_template = 'Found useless `except` case'
     code = 329
+
+
+class UselessOperatorsViolation(ASTViolation):
+    """
+    Forbids the use of unnecessary operators in your code.
+
+    You can write: ``5.4`` and ``+5.4``. There's no need to use the second
```

Comment:
Can you please rephrase it to be more readable? Since we only support `+` for now, you can only mention `+` operators.